.github/scripts/generate_manifests.bash

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Script for the initial generation of manifest.yaml files in inputs directories
+
+traverse_dir() {
+  local dir="$1"
+  local contains_subdir=false
+
+  # Loop through the directory contents
+  for file in "$dir"/*; do
+    if [ -d "$file" ]; then
+      contains_subdir=true
+      traverse_dir "$file"
+    fi
+  done
+
+  # If the directory does not contain any subdirectories...
+  if [ "$contains_subdir" = false ]; then
+    echo "Working on $dir:"
+    # Find all the files that we want to turn into manifests
+    files=$(find "$dir" -maxdepth 1 -type f -printf '%f ')
+    if [ -n "$find" ]; then
+      # If there are some files, add them using yamf
+      cd "$dir" || exit 2
```

Comment:
I think, based on the code above, `cd "$dir"` would never fail here, making the `|| exit 2` part redundnt.

Reply:
Very true. Was just a shellcheck warning so I added it anyway :D